client/index.html

```diff
@@ -24,31 +24,61 @@ <h1>Code Runner</h1>
 			<div class="content"> 
 			<form id="mainForm">
 				<div class="form-group row">
-					<label for="assgn" class="col-sm-2 col-form-label">Assignment:</label>
-					<div class="col-sm-5">
-						<input type="file" id="file" name="file" class="form-control" onchange="verifyFiles(this)">
+					<span class="col-sm-10 col-form-label" id="note"><small>Note: Enter working directory relative to the assignment root directory. Example - For <b>assignment_1.tar</b> root directory is <b>assignment_1</b>.
```

Comment:
Before the example, maybe mention that **Submitted tar/zip files are extracted into a directory with the same name as the submitted file.**

Reply:
Okay